corehq/apps/export/models/new.py

```diff
@@ -235,42 +293,93 @@ def _generate_schema_from_xform(xform, langs, appVersion):
 
         return schema
 
+
+class CaseExportDataSchema(ExportDataSchema):
+
     @staticmethod
-    def _merge_schema(schema1, schema2):
-        """Merges two ExportDataSchemas together
+    def generate_schema_from_builds(domain, case_type):
+        """Builds a schema from Application builds for a given identifier
 
-        :param schema1: The first ExportDataSchema
-        :param schema2: The second ExportDataSchema
-        :returns: The merged ExportDataSchema
+        :param domain: The domain that the export belongs to
+        :param unique_form_id: The unique identifier of the item being exported
+        :returns: Returns a ExportDataSchema instance
         """
+        app_build_ids = get_all_app_ids(domain)
+        all_case_schema = CaseExportDataSchema()
 
-        schema = ExportDataSchema()
+        for app_doc in iter_docs(Application.get_db(), app_build_ids):
+            app = Application.wrap(app_doc)
+            case_property_mapping = get_case_properties(
+                app,
+                [case_type],
+                include_parent_properties=False
+            )
+            case_schema = CaseExportDataSchema._generate_schema_from_case_property_mapping(
+                case_property_mapping,
+                app.version,
+            )
+            case_history_schema = CaseExportDataSchema._generate_schema_for_case_history(
+                case_property_mapping,
+                app.version,
+            )
 
-        def resolvefn(group_schema1, group_schema2):
-            group_schema = ExportGroupSchema(
-                path=group_schema1.path,
-                last_occurrence=max(group_schema1.last_occurrence, group_schema2.last_occurrence),
+            all_case_schema = CaseExportDataSchema._merge_schemas(
+                all_case_schema,
+                case_schema,
+                case_history_schema
             )
-            items = _merge_lists(
-                group_schema1.items,
-                group_schema2.items,
-                keyfn=lambda item: '{}:{}'.format(_list_path_to_string(item.path), item.doc_type),
-                resolvefn=lambda item1, item2: item1.__class__.merge(item1, item2),
-                copyfn=lambda item: item.__class__(item.to_json()),
+
+        return all_case_schema
+
+    @staticmethod
+    def _generate_schema_from_case_property_mapping(case_property_mapping, appVersion):
```

Comment:
just as the other one was for Case History, this one is for the main case property table:

<img width="1201" alt="screen shot 2016-01-27 at 10 24 39 am" src="https://cloud.githubusercontent.com/assets/918514/12618031/3184705c-c4e0-11e5-92b7-191dcc6d8cdd.png">